pkg/reconciler/revision/resources/deploy.go

```diff
@@ -210,6 +227,10 @@ func makePodSpec(rev *v1.Revision, cfg *config.Config) (*corev1.PodSpec, error)
 		}
 	}
 
+	if cfg.Deployment.Affinity == deploymentconfig.PreferSpreadRevisionOverNodes && cfg.Features.PodSpecAffinity == apiconfig.Disabled {
```

Comment:
Hmm, ok. I misunderstood then.  I'll take a look at the code again and adjust it

Reply:
Hm. IMHO, isn't the current config variation seems a bit unintuitive:

- feature: enabled = we don't set it
- feature: disabled + manually set to none = we don't set it
- feature: disabled + not manually disabled = we set it

Could we rename the flag to "defaultAffinity" and apply it, regardless of the feature config is enabled/disabled (so that is clear that we apply this by default if not specified otherwise). The user could then still enable the feature flag and override it on a service basis.

Reply:
> Could we rename the flag to "defaultAffinity" and apply it

Changing it to `defaultAffinity` makes sense - maybe `defaultAffinityType` since we're not setting the affinity but the type we want.

> regardless of the feature config is enabled/disabled (so that is clear that we apply this by default if not specified otherwise). The user could then still enable the feature flag and override it on a service basis.

Thats my intent and what I'm asking for.

Reply:
Renamed the config to defaultAffinityType